config/config.yaml

```diff
@@ -476,6 +476,8 @@ defaults:
     k8s:
       namespace: clusters-service
       serviceAccountName: clusters-service
+    azureRuntimeConfig:
+      tlsCertificatesIssuer: ""
```

Comment:
Shouldn't this be "Self"?

Reply:
we can set it to the OneCertV2-Public one as basically all clouds but `dev` will use that value.

Reply:
does dev include both shared dev and personal dev? @geoberle .

What I would like is:

* dev and personal dev use Self
* the other ones use OneCertV2-Public

Reply:
@miguelsorianod 
having `OneCertV2-Public` in the top-level defaults is ok.
the `dev` cloud refers to the RH tenant. override it to `Self` defined there is required as there is no onecert outside of MSFT

@jubittajohn 
exactly